var/spack/repos/builtin/packages/boost/package.py

```diff
@@ -135,6 +135,18 @@ class Boost(Package):
     depends_on('zlib', when='+iostreams')
     depends_on('py-numpy', when='+numpy', type=('build', 'run'))
 
+    # Coroutine, Context, Fiber, etc., are not straightforward.
+    conflicts('+context', when='@:1.50.99')  # Context since 1.51.0.
+    conflicts('cxxstd=98', when='+context')  # Context requires >=C++11.
+    conflicts('+coroutine', when='@:1.52.99')  # Context since 1.53.0.
+    conflicts('~context', when='+coroutine')  # Coroutine requires Context.
+    conflicts('+fiber', when='@:1.61.99')  # Context since 1.62.0.
```

Comment:
Is this a typo? "# Context since 1.62.0", I would have assumed this would read "# Fiber since 1.62.0"